docs/book/src/tasks/experimental-features/cluster-class/operate-cluster.md

```diff
@@ -1,8 +1,18 @@
 # Operating a managed Cluster
 
-## Upgrade the Cluster
+The `spec.topology` field added to the Cluster object as part of ClusterClass allows changes made on the Cluster to be propagated across all relevant objects. This means the Cluster object can be used as a single point of control for making changes to objects that are part of the Cluster, including the ControlPlane and MachineDeployments. 
 
-The `spec.topology` field added to the Cluster object as part of ClusterClass allows changes made on the Cluster to be propagated across all relevant objects. This turns a Kubernetes cluster upgrade into a one-touch operation.
+A managed Cluster can be used to:
+* [Upgrade a Cluster](#upgrade-a-cluster)
+* [Scale a ControlPlane](#scale-a-controlplane)
+* [Scale a MachineDeployment](#scale-a-machinedeployment)
+* [Add a MachineDeployment](#add-a-machinedeployment)
+* [Use variables in a Cluster](#use-variables)
+* [Rebase a Cluster to a different ClusterClass](#rebase-a-cluster)
+
+
+## Upgrade a Cluster
+Using a managed topology the operation to upgrade a Kubernetes cluster is a one-touch operation.
```

Comment:
Hm we have a mismatch here (two lines below). The quickstart currently deploys a v1.23.3 cluster. 

As the quickstart aims to use the latest Kubernetes release for which kind images exist, it will probably almost never work as the source version for an upgrade example.